wire-ios/Wire-iOS/Sources/UserInterface/Settings/AccentColorPicker.swift

```diff
@@ -16,49 +16,50 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Inject
 import SwiftUI
 import WireCommonComponents
 import WireDesign
 import WireFoundation
 import WireSyncEngine
 
+#if canImport(Inject)
```

Comment:
This is not needed.

Reply:
It might not be needed for Inject to be not included in Release builds, however, it still allows us to compile this code without having Inject linked.